src/assets/image.js

```diff
@@ -0,0 +1,13 @@
+export { default as avatar1 } from './avatar1.jpeg';
+export { default as avatar2 } from './avatar2.jpeg';
+export { default as avatar3 } from './avatar3.jpeg';
+export { default as avatar4 } from './avatar4.jpeg';
+export { default as bgTexture } from './bg-texture.png';
+export { default as meAbout } from './me-about.jpeg';
+export { default as me } from './me.png';
+export { default as portfolio1 } from './portfolio1.png';
```

Comment:
Image could be descriptive. Avatar 1, 2, 3... is not looks good. Same for other images.